test/remote/base_path_tests.js

```diff
@@ -2,13 +2,14 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+process.env.PUBLIC_URL = 'http://127.0.0.1:9000/auth'
```

Comment:
I had to move this up so that it gets set before any other attempts to require the config module (i.e. that I introduced in the token module).